examples/data-objects/codemirror/src/index.ts

```diff
@@ -15,41 +14,41 @@ import {
     buildRuntimeRequestHandler,
 } from "@fluidframework/request-handler";
 import { defaultRouteRequestHandler } from "@fluidframework/aqueduct";
-
+import { RuntimeFactoryHelper } from "@fluidframework/runtime-utils";
 import { fluidExport as smde } from "./codemirror";
 
-class CodeMirrorFactory implements IRuntimeFactory {
-    public get IRuntimeFactory() { return this; }
+const defaultComponentId = "default";
+const defaultComponent = "@fluid-example/smde";
```

Comment:
defaultComponent -> defaultComponentType

Reply:
Or rather, smdeType, or even better - remove this variable and use smde.type inline - it's a type of factory used.
Same in other examples - unless there is very good reason, we should never overwrite / give custom type to a factory, but rather use the one provided by factory itself. I'd love to go even further and say that type as argument should be removed as factory has it.